### DISCUSSION TITLE
FYI: Ionos also launched a cluster-api machine-provider for proxmox

### DISCUSSION THREAD

**Discussion by 3deep5me** [k8s-proxmox/cluster-api-provider-proxmox#163] [General]
May worth a look:
https://github.com/ionos-cloud/cluster-api-provider-proxmox

I don't know how the projects differ especially of maturity. 
But I didn't want to withhold the news from you.

**Comment by sp-yduck** [comment]
hmmm interesting... 
should we propose to merge our repo ?

**Reply by 3deep5me** [reply]
I would like to see how good there provider work but at the moment i ran into [2#](https://github.com/ionos-cloud/cluster-api-provider-proxmox/issues/22). 
What i can say so far from a user perspective i miss these feature in the ionos provider:
- dhcp for nodes 
- auto download of images
- the options of the qume-scheduler
- no proxmox-ccm

On the other hand i like these things on the ions provider:
- development with tilt
- flavors for cni for simpler use
- creation with vmtemplate-id 
- simpler crd's (but properly far less functions)

All in all on the first look of a user our/your provider looks better but i like the fact that is backed by a company.

Why not ask them to merge our/your provider or adopt big parts of it?
But i would definitely love to adopt things like sonar or documentation and tilt. 

At the end I don't think it helps anyone to develop two providers.

**Reply by sp-yduck** [reply]
yeah I don't want to waste my time on duplicate work. let's ask them firstly

fyi I opened an issue on their repo
https://github.com/ionos-cloud/cluster-api-provider-proxmox/issues/24

**Reply by 3deep5me** [reply]
I fully agree 💯

**Reply by wikkyk** [reply]
Hi. Just FYI, I replied to https://github.com/ionos-cloud/cluster-api-provider-proxmox/issues/24 - feel free to move the discussion there. In short, yes, we're interested in merging.
